static/src/javascripts/projects/common/utils/url.js

```diff
@@ -1,8 +1,7 @@
 define([
     'common/utils/detect',
-    'common/utils/mediator',
-    'lodash/utilities/identity'
-], function (detect, mediator, identity) {
+    'common/utils/mediator'
```

Comment:
It is indeed not a great design. Looks like it isn't used anywhere. Let's scratch it!